### PR TITLE
Prop for inline styling of slide component

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
Hello, and thanks for the awesome component you have made!

We would need to be able to style the slide component, so I made a modification to the source which allows user to pass inline styles for the slide as props.

BR,
Ville Miekk-oja

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -261,7 +268,7 @@ class SwipeableViews extends React.Component {
       }
 
       return (
-        <div ref={ref} style={styles.slide}>
+        <div ref={ref} style={Object.assign(styles.slide, slideStyle)}>
```
Comment (oliviertassinari) [inline src/index.js]: Can we move the `Object.assign` outside of the loop, like just before the `childrenToRender`?
Also use `Object.assign({}, styles.slide, slideStyle)` instead.

**Comment by oliviertassinari** [comment]
@VilleMiekkoja Thanks for opening this PR :+1:.

File: src/index.js (hunk 1 above)
Comment (VilleMiekkoja) [inline src/index.js]: Yes, good idea so we only need to do it once.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,7 @@ export default MyComponent;
 | onSwitching | Function(index) |  | This is callback prop. It's called by the component when the slide switching. This is useful when you want to implement something corresponding to the current slide position. |
 | resistance | bool | false | If true, it will add bounds effect on the edges. |
 | style | object | {} | This is the inlined style that will be applied on the root component. |
+| styleSlide | object | {} | This is the inlined style that will be applied on the slide component. |
```
Comment (oliviertassinari) [inline README.md]: `slideStyle`

**Comment by oliviertassinari** [comment]
@VilleMiekkoja Thanks!

**Comment by VilleMiekkoja** [comment]
No problem, thanks for you!

BR,
Ville

perjantai 4. maaliskuuta 2016 Olivier Tassinari <notifications@github.com>
kirjoitti:

**Pull request merged**